sdk/python/kfp/v2/compiler_cli_tests/test_data/pipeline_with_task_final_status.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Pipeline using ExitHandler with PipelineTaskFinalStatus."""
+
+from kfp.v2 import dsl
+from kfp.v2 import compiler
+from kfp.v2.dsl import component
+
+from kfp.v2.dsl import PipelineTaskFinalStatus
+
+
+@component
+def exit_op(user_input: str, status: PipelineTaskFinalStatus):
```

Comment:
nit: I think this should be `dsl.PipelineTaskFinalStatus` per [Google Python Style Guide 2.2 Imports](https://google.github.io/styleguide/pyguide.html#22-imports).

I haven't been entirely consistent about this in my own PRs, FWIW.